lib/perl/Genome/Qc/Config.pm

```diff
@@ -0,0 +1,36 @@
+package Genome::Qc::Config;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::Qc::Config {
+    has => [
```

Comment:
For now we could make this
`is => 'UR::Value',
id_by => [
  name => { is => 'String' },
]`.

That way we could still `Genome::Qc::Config->get(name => 'turkey')` and get an object back (and could thus wire up everything instead of leaving it commented out).  Once there's a real DB-backed entity for that name, we swap out the class definition and everything still works!

[This was one of the changes in one of the branches we were discussing.]

In the context of such a change, does the `Factory` still need to exist?